Tests/Fixtures/Generated/XCAssets/swift3/all-customName.swift

```diff
@@ -2,15 +2,20 @@
 // Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
 
 #if os(macOS)
-  import AppKit.NSImage
-  internal typealias XCTColor = NSColor
-  internal typealias XCTImage = NSImage
-#elseif os(iOS) || os(tvOS) || os(watchOS)
-  import UIKit.UIImage
-  internal typealias XCTColor = UIColor
-  internal typealias XCTImage = UIImage
+  import AppKit
+#elseif os(iOS)
+  import ARKit
+  import UIKit
+#elseif os(tvOS) || os(watchOS)
+  import UIKit
```

Comment:
iirc in a previous commit before force-push, you added ARKit under `if is(tvOS/watchOS)` too? So I'm curious, did our unit tests, especially the rake trying to compile generated code, properly caught that `import ARKit` didn't work on tvOS/watchOS, before you fixed that? If not (= if you did just realize that by yourself not thanks to tests) we should make sure our `rake output:compile` catches that in the future.

Reply:
I'm sure it never was under watchOS/tvOS from the start. I checked apple docs before I initially started to modify the template. Maybe it was some artefact of a bad merge?

Do keep in mind that the swift3 templates are no longer compiled, so CI can't catch it in this case. The swift 4 output is compiled on all 4 platforms.

Reply:
Yeah maybe it was also me misreading the diff on my last skimming then 😅

Reply:
(Note that my comment wasn't specially on swift3 only, I didn't even realize that I made that comment on Swift 3 template specifically, it just happened to be the first instance where I wondered about that during my review ^^)